superset/databases/schemas.py

```diff
@@ -1195,3 +1195,27 @@ class OAuth2ProviderResponseSchema(Schema):
     class Meta:  # pylint: disable=too-few-public-methods
         # Ignore unknown fields that might be sent by the OAuth2 provider
         unknown = EXCLUDE
+
+
+class QualifiedTableSchema(Schema):
+    """
+    Schema for a qualified table reference.
+
+    Catalog and schema can be ommited, to fallback to default values. Table name must be
+    present.
+    """
+
+    table = fields.String(
```

Comment:
I know I used the term `table` in the `Table` class, but I wonder if this should be name given it's referring to the name of the table. This is also consistent with your description on line 1210.

Reply:
Yeah, I think the current terminology is confusing, there's a lot of `table.table` in the codebase. Let me change it.